docs/compute-element/htcondor-ce-overview.md

```diff
@@ -131,7 +131,7 @@ installation, there are two main ways to customize your CE (if you decide any cu
 How Security Works
 ------------------
 
-In the OSG, communication is secured between various parties using a combination of PKI infrastructure involving
+Communication is secured between various parties using a combination of PKI infrastructure involving
```

Comment:
I feel like starting with “Communication” is too broad. In the spirit of the original, maybe “Among OSG services, communication …”?

Reply:
So "OSG services" is allowed?

Reply:
Assuming so, then fixed using your phrasing.